plugins/mapper-annotated-text/build.gradle

```diff
@@ -9,23 +9,18 @@ import org.elasticsearch.gradle.internal.info.BuildParams
  */
 apply plugin: 'elasticsearch.internal-yaml-rest-test'
 apply plugin: 'elasticsearch.yaml-rest-compat-test'
-apply plugin: 'elasticsearch.internal-cluster-test'
 
 esplugin {
   description 'The Mapper Annotated_text plugin adds support for text fields with markup used to inject annotation tokens into the index.'
   classname 'org.elasticsearch.index.mapper.annotatedtext.AnnotatedTextPlugin'
 }
 
 if (BuildParams.isSnapshotBuild() == false) {
-  tasks.named("internalClusterTest").configure {
+  tasks.named("test").configure {
```

Comment:
This has been annoying me for *ages*, thanks for fixing

Reply:
:bow: